.github/workflows/sync_docs.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: Sync docs from Discourse
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: 50 00 * * *
```

Comment:
It is a mongodb-k8s time: https://docs.google.com/spreadsheets/d/1crHqnVtuGHsNIrOYGsfkIiGUA7peJNFNOqMkvTKVvd4/edit#gid=0

PG K8s is 00:30